### DISCUSSION TITLE
Lottie/Rive animations in Toga

### DISCUSSION THREAD

**Discussion by tct123** [beeware/toga#2892] [Ideas]
Hello, will it be possible to play Lottie/Rive animations in Toga?

**Comment by freakboy3742** [comment]
I've never heard of Lottie or Rive before, so the short answer is I don't know.

The longer answer is that Toga doesn't currently have support for any sort of animated image. 

However, if there is native platform support for these image types (which a search seems to suggest exists), then there are ways to access that native platform capability from within a Toga app. The native GUI layers for any given widget are accessible; and you can invoke (almost) any native API over the bridging layers that Toga uses (Rubicon, Chaquopy etc).
